telebot/modules/regex.py

```diff
@@ -0,0 +1,65 @@
+from re import compile, search
+from subprocess import Popen, PIPE, check_output, CalledProcessError, STDOUT
+
+from telegram import MAX_MESSAGE_LENGTH, Update
+from telegram.ext import CallbackContext, MessageHandler, Filters
+
+from telebot import dispatcher
+from telebot.functions import bot_action
+from telebot.modules.db.exceptions import get_exceptions_for_chat
+
+
+@bot_action("regex")
+def regex(update: Update, context: CallbackContext):
+    """
+    Replace text using sed and regex
+    :param update: object representing the incoming update.
+    :param context: object containing data about the command call.
+    """
+    if "regex" in get_exceptions_for_chat(update.effective_chat.id):
+        return
+
+    if not update.effective_message.reply_to_message:
+        update.effective_message.reply_text("Gimme text to replace stuff in, baka!")
+        return
+
+    # make sure `s` command is properly terminated
+    command = update.effective_message.text
+    delimiter = command[1]
```

Comment:
Shouldn't we check what it is, exactly?

if command [1] in ('/',',''|') kinda

Reply:
I mean, we're using a message handler with a regex filter to trigger the command anyway, so won't this check be a bit too redundant?

Reply:
Missed that. Nvm

Reply:
Missed that. Nvm